widgets/SapOData-batchsource.json

```diff
@@ -0,0 +1,83 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name": "SAP OData",
+  "configuration-groups": [
+    {
+      "label": "General",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "OData Service URL",
+          "name": "url"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Resource Path",
+          "name": "resourcePath",
+          "plugin-function": {
+            "label": "Get Schema",
+            "widget": "outputSchema",
+            "output-property": "schema",
+            "omit-properties": [
+              {
+                "name": "schema"
+              }
+            ]
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Query",
+          "name": "query"
+        }
+      ]
+    },
+    {
+      "label": "Credentials",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Username",
+          "name": "username"
+        },
+        {
+          "widget-type": "password",
+          "label": "Password",
+          "name": "password"
+        }
+      ]
+    }
+  ],
+  "emit-errors": true,
```

Comment:
from the code it doesn't look like it emits errors, so this shouldn't be set.

Reply:
Done.